src/app/pages/TokenDashboardPage/TokenTypeCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { SnapshotCard } from '../../components/Snapshots/SnapshotCard'
+import { COLORS } from '../../../styles/theme/colors'
+import { useRequiredScopeParam } from '../../hooks/useScopeParam'
+import { useTokenInfo } from './hook'
+import { useLoaderData } from 'react-router-dom'
+import { EvmTokenType } from '../../../oasis-indexer/api'
+import { TFunction } from 'i18next'
+import { exhaustedTypeWarning } from '../../../types/errors'
+
+export const getTokenTypeName = (t: TFunction, type: EvmTokenType): string => {
+  switch (type) {
+    case 'ERC20':
+      return t('account.ERC20')
+    default:
+      void exhaustedTypeWarning('Unknown token type', type)
```

Comment:
Is `void` an intentional keyword here? I don't know what it does

Reply:
Yes it is intentional. It explicitly declares that we are not interested in the promise returned by `exhaustedTypeWarning()`. Otherwise some code quality tools will sometimes complain about the ignored promise.

The bigger question is, why is `exhaustedTypeWarning()` even an async function? Does it have to be?

Reply:
:O it's async?

Reply:
Definitely does not need to be